clients/cloud/nodejs/consumer.js

```diff
@@ -28,8 +28,8 @@ function createConsumer(config, onData) {
     'bootstrap.servers': config['bootstrap.servers'],
     'sasl.username': config['sasl.username'],
     'sasl.password': config['sasl.password'],
-    'security.protocol': 'SASL_SSL',
-    'sasl.mechanisms': 'PLAIN',
+    'security.protocol': config['security.protocol'],
```

Comment:
Where is it getting `security.protocol` and `sasl.mechanisms` from?  It's not currently in the config file.

Reply:
ah i didn't notice those were removed from here https://github.com/confluentinc/configuration-templates/blob/master/clients/cloud/librdkafka.config. is there any reason to not have `sasl.mechanism` and `security.protocol` in the config files? We have them in the java-type ones, and it would help make the examples more consistent

Reply:
@laura-brouckman  The examples repo used to have a mix (e.g. sometimes `security.protocol` was provided in the client config and sometimes in the code example itself) and because it was this unfortunate mix, that caused confusion.  

I see that you fixed that in your PR.

tl;dr my apologies, you have it the right way, and I will fix https://github.com/confluentinc/configuration-templates/blob/master/clients/cloud/librdkafka.config

Reply:
Another great example of why implementing this single source of truth is so helpful.

Reply:
@laura-brouckman , @rspurgeon  https://github.com/confluentinc/configuration-templates/pull/2